oauthenticator/github.py

```diff
@@ -141,10 +208,23 @@ def _check_organization_whitelist(self, org, username, access_token):
                 return True
         return False
 
+    @gen.coroutine
+    def _get_user_organizations(self, access_token):
+        http_client = AsyncHTTPClient()
+        headers = _api_headers(access_token)
+        next_page = "https://%s/user/orgs" % (GITHUB_API)
+        orgmap = {}
+        while next_page:
+            req = HTTPRequest(next_page, method="GET", headers=headers)
+            resp = yield http_client.fetch(req)
+            resp_json = json.loads(resp.body.decode('utf8', 'replace'))
+            next_page = next_page_from_links(resp)
+            for entry in resp_json:
+                orgmap[entry["login"]] = entry["id"]
```

Comment:
Because I'm using the ID to construct group ID mappings on the back end.  As with the GH ID, this keeps my group IDs the same, which then lets me create globally-consistent user groups from github org membership.